ibm_db2/requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 -e ../datadog_checks_dev
-ibm_db==3.0.1
+ibm_db==3.0.1; python_version < "3.0" 
```

Comment:
Seems it's still trying to build 3.1.0 with python 2

```
Collecting ibm_db
  Downloading ibm_db-3.1.0.tar.gz (797 kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /opt/datadog-agent/embedded/bin/python2 /opt/datadog-agent/embedded/lib/python2.7/site-packages/pip-20.3.3-py2.7.egg/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpccYTex
       cwd: /tmp/pip-install-BSohTv/ibm-db
  Complete output (18 lines):
  Detected 64-bit Python
  Downloading https://public.dhe.ibm.com/ibmdl/export/pub/software/data/db2/drivers/odbc_cli/linuxx64_odbc_cli.tar.gz
  Traceback (most recent call last):
    File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pip-20.3.3-py2.7.egg/pip/_vendor/pep517/_in_process.py", line 280, in <module>
      main()
    File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pip-20.3.3-py2.7.egg/pip/_vendor/pep517/_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/pip-20.3.3-py2.7.egg/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-VfOIaC/overlay/lib/python2.7/site-packages/setuptools/build_meta.py", line 146, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-VfOIaC/overlay/lib/python2.7/site-packages/setuptools/build_meta.py", line 127, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-VfOIaC/overlay/lib/python2.7/site-packages/setuptools/build_meta.py", line 142, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 240, in <module>
      file_stream = BytesIO(request.urlopen(url, context=context).read())
  TypeError: initial_value must be unicode or None, not str
  ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/datadog-agent/embedded/bin/python2 /opt/datadog-agent/embedded/lib/python2.7/site-packages/pip-20.3.3-py2.7.egg/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpccYTex Check the logs for full command output.
```

Reply:
Yes, I'm trying to find a way to override this

Reply:
I had to add the installation of different version markers in the start commands